src/Symfony/Component/Form/DataTransformerInterface.php

```diff
@@ -54,8 +57,10 @@ interface DataTransformerInterface
      * process that value.
      *
      * @param mixed $value The value in the original representation
+     * @psalm-param T|null $value
```

Comment:
We don't need to use the `@psalm-` prefix, isn't it? (same question everwhere)

Reply:
Done